SAMv1.tex

```diff
@@ -1204,6 +1204,48 @@ \subsection{C source code for computing bin number and overlapping bins}\label{s
 \end{verbatim}
 }
 
+\subsection{Splitting BAM}\label{sec:code}
+A BAM file can be processed in parallel by conceptually dividing the file into
+splits (typically of a fixed, but arbitrary, number of bytes) and for each
+split processing alignments from the first known alignment after the split
+start up to the first known alignment of the next split.
+
+A splitting BAM index is a linear index of virtual file offsets of alignment
+start positions. The index must contain the virtual file offset for the first
+alignment, and a virtual file offset for the overall length of the BAM
+file.\footnote{In the unlikely event the BAM file has no alignment records,
+the index will consist of a single entry for the overall length of the
+BAM file.} It does not need to contain a virtual file offset for every
+alignment, merely a subset. A granularity of $n$ means that an offset is
```

Comment:
Do we want to allow indication of approximate number of records per offset?  Or is that just making things unnecessarily complicated?  At the cost of increasing the index size we could include the number of records in each section in the index.  Instead of `[offset]`, we'd have `[offset, number of records until next offset]`  That might be useful for tools deciding how to split the file.